### ISSUE TITLE
Create content for Visual Design Style guide

### ISSUE THREAD

**Issue by adam-little** [cagov/design-system#255]
The design team needs to document our approach to visual design for the design system. Draft content is [here](https://docs.google.com/document/d/17MtpWf1rZIl_Ge05DEQqIsd_NxBYha6krjRBhSUhdXg/edit#heading=h.vvazwqrdhr7l).

Outline for MVP:
- [x] Approach (assigned to @adam-little)
- [x] Color
- [x] Typography (assigned to @timothyshambra) 
- [x] Imagery
- [x] Accessibility considerations (assigned to @kmbrlygl)

Ideas for post-MVP launch:

- Icons: Link to icons in SVG format
- Data viz: Guidelines for data visualizations
- Tools for designers: Figma UI kit

**Comment by adam-little** [comment]
In our draft Google Doc, I just added a section for high-level guidelines. I believe these can build on the Design System Principles and start to articulate rational for visual design choices in lieu of having a brand strategy.

**Comment by adam-little** [comment]
@kmbrlygl here are some thoughts re: visual design accessibility considerations

- According to https://webstandards.ca.gov/accessibility/, state entities are to align with Web Content Accessibility Guideline (WCAG) 2.1 Level AA Standards in addition to the requirements of Section 508 (29 U.S.C. 794d)
- What would a visual designer need to know about those standards?
- This is a good resource. Should we link to this from our page? https://accessibility.digital.gov/visual-design/getting-started/
- There's no need for us to recreate a guide like the one linked above, so what would we highlight for the design system?

**Comment by adam-little** [comment]
I updated the approach section with input from Kimberly and copy-editing by Michael.

If anyone has additional comments or suggestions, please leave a comment in [the Google Doc](https://docs.google.com/document/d/17MtpWf1rZIl_Ge05DEQqIsd_NxBYha6krjRBhSUhdXg/edit#).

**Comment by kmbrlygl** [comment]
@adam-little I made edits and suggestions to the Accessibility Considerations section. I have a few comments in there to look over. There are some overlaps with content design and development which we could just keep. Just trying to make it more focused on visual design. 

I added a blurb specifically mentioning the WCAG standards and Section 508. I saw that we mention WCAG on the homepage at a higher level, though not Section 508. I'm not sure if this is mentioned anywhere else so I figured I'd reiterate it just in case. 

Let me know what you think and if you have any questions.
